src/material_library.cpp

```diff
@@ -135,6 +136,23 @@ void pyne::MaterialLibrary::write_json(const std::string& filename) {
   f.close();
 }
 
+void pyne::MaterialLibrary::write_openmc(const std::string& filename) const
+{
+  std::ofstream f(filename.c_str());
+  // write header
+  f << "<?xml version=\"1.0\"?>\n";
+  f << "<materials>\n";
```

Comment:
Mmm, good point. I think this function is fine since the `<materials`> tag isn't indented, but the `<material`> tags had some indentation that was off.

I've corrected those and added an argument to the `Material::openmc` method for the indention level (1 by default assuming the materials are being written directly into a standard materials XML file w/o modification).

Changes in 12eeab0e74cebea0e44191a7eed2db612e8cf7cb.